src/__test__/components/data-exploration/heatmap/HeatmapPlot.test.jsx

```diff
@@ -62,20 +63,14 @@ jest.mock('lodash/sampleSize', () => ({
   __esModule: true,
 }));
 
-// // Worker responses are fetched from S3, so these endpoints are added to fetchMock
-// // the URL for the endpoints are generated by the functions passed to mockETag above
+enableFetchMocks();
+
 const mockWorkerResponses = {
   '5-marker-genes': () => Promise.resolve(_.cloneDeep(markerGenesData5)),
   '2-marker-genes': () => Promise.resolve(_.cloneDeep(markerGenesData2)),
 };
 
-let storeState = null;
-const loadAndRenderDefaultHeatmap = async () => {
-  storeState = makeStore();
-
-  storeState.dispatch(loadProcessingSettings(experimentId));
-  storeState.dispatch(loadBackendStatus(experimentId));
-
```

Comment:
why are these actions not necessary anymore?

Reply:
Because it is already being called inside the component. We do not need to preload it into the store, the component will do it as part of its initialization.